packages/netsuite-adapter/test/change_validators/unreferenced_file_addition.test.ts

```diff
@@ -0,0 +1,165 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import { ElemID, InstanceElement, ObjectType, ReferenceExpression, toChange } from '@salto-io/adapter-api'
+import { EMAIL_TEMPLATE, FILE, NETSUITE } from '../../src/constants'
+import unreferencedFileAddition from '../../src/change_validators/unreferenced_file_addition'
+
+describe('unreferenced file addition validator', () => {
+  describe('script file', () => {
+    const scriptFileNacl = new InstanceElement('scriptFileNacl', new ObjectType({ elemID: new ElemID(NETSUITE, FILE) }))
+    const scriptReferenceElement = new InstanceElement('scriptElemWithReference', new ObjectType({ elemID: new ElemID(NETSUITE, 'suitlet') }),
+      { defaultfunction: 'svda',
+        scriptfile: new ReferenceExpression(
+          scriptFileNacl.elemID
+        ) })
```

Comment:
styling nit - new line after `{`, especially when the whole object is in several lines
```suggestion
      {
        defaultfunction: 'svda',
        scriptfile: new ReferenceExpression(scriptFileNacl.elemID),
      })
```